tostadas/1.0.0/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM nfcore/base:1.13.3
+LABEL authors="Cole and Ankush Gupta" \
+      description="Docker image containing all software requirements for mamba env"
```

Comment:
Could you add more labels? These are from https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile

```
LABEL base.image="ubuntu:focal"
LABEL dockerfile.version="1"
LABEL software="SoftwareName"
LABEL software.version=$SOFTWARENAME_VER
LABEL description="This software does X, Y, AND Z!"
LABEL website="https://github.com/StaPH-B/docker-builds"
LABEL license="https://github.com/StaPH-B/docker-builds/blob/master/LICENSE"
LABEL maintainer="FirstName LastName"
LABEL maintainer.email="my.email@email.com"
```